src/SFA.DAS.RoatpOversight.Domain/ModerationStatus.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.RoatpOversight.Domain
+{
+    public static class ModerationStatus
+    {
+        public const string Pass = "Pass";
+    }
```

Comment:
Hi, can you update this class to be:   public class ModerationReviewStatus
    {
        public const string Fail = "Fail";
        public const string Pass = "Pass";
    }